src/rpc/blockchain.cpp

```diff
@@ -1161,6 +1163,197 @@ static RPCHelpMan pruneblockchain()
     };
 }
 
+static RPCHelpMan dumpcoinstats()
+{
+    return RPCHelpMan{"dumpcoinstats",
+        "Dumps content of coinstats index to a CSV file. The resulting file contains the following columns:\n"
+        "- height\n"
+        "- block_hash\n"
+        "- timestamp\n"
+        "- mediantime\n"
+        "- version\n"
+        "- (total_)txs - Number of transactions included\n"
+        "- (total_)txouts - Number of transaction outputs created\n"
+        "- (total_)amount_spent - Amount of inputs spent\n"
+        "- (total_)amount_created - Amount of coins added to the UTXO set\n"
+        "- (total_)coinbase_amount - Coinbase subsidy amount\n"
```

Comment:
"Amount claimed in coinbase transaction." (see e.g. block 409,008 which claimed a whopping 17K BTC.

Reply:
Done